.github/workflows/ci.yaml

```diff
@@ -0,0 +1,52 @@
+on:
+  push:
+    branches:
+      - master, github-action
```

Comment:
Why do we limit which branches we execute CI on? Why not just do a blanket `on: push`?

Reply:
I agree, actually. Traditionally we only build PRs and pushes to master, however.

Reply:
I was just doing it on PRs and master to start to see how much of our existing minutes budget would get used up first.